src/librbd/cache/pwl/Types.h

```diff
@@ -177,7 +177,8 @@ const uint64_t MIN_POOL_SIZE = DEFAULT_POOL_SIZE;
 const uint64_t POOL_SIZE_ALIGN = 1 << 20;
 constexpr double USABLE_SIZE = (7.0 / 10);
 const uint64_t BLOCK_ALLOC_OVERHEAD_BYTES = 16;
-const uint8_t RWL_POOL_VERSION = 1;
+const uint8_t RWL_POOL_VERSION = 2;
+const uint64_t SSD_POOL_VERSION = 1;
```

Comment:
this must be used for layout_version, which IIUC is either 0,1 or 2; should we keep same datatype then?

Reply:
@CongMinYin I meant using the smaller size data type: `uint8_t` since it can accommodate all conditions

Reply:
Modified. As a constant, 8 bytes is enough.